_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -2130,6 +2130,26 @@ struct FindNonSkip
 
     mfxU16 m_skipFrameMode;
 };
+mfxStatus FixForcedFrameType (DdiTask & newTask, mfxU32 frameOrder)
```

Comment:
passing ```DdiTask``` is too much for this function, I think it should be ```Pair<mfxU8>& type``` instead

Reply:
where are a lot similar functions in encoder. Let's it be in one style.